Explorer/Assets/DCL/Multiplayer/Connections/Messaging/MessageWrap.cs

```diff
@@ -51,9 +54,11 @@ public async UniTaskVoid SendAndDisposeAsync(CancellationToken cancellationToken
 
             using var packetWrap = multiPool.TempResource<Packet>();
             packetWrap.value.ClearMessage();
+            packetWrap.value.ProtocolVersion = supportedVersion;
             WritePayloadToPacket(packetWrap.value);
             using MemoryWrap memory = memoryPool.Memory(packetWrap.value);
             packetWrap.value.WriteTo(memory);
+            packetWrap.value.ProtocolVersion = 0;
```

Comment:
Why do you reset the ProtocolVersion here? Is some kind of disposal before returning to pool?

Reply:
Yes